test/extern/fnPtrs/passBadFnsToCChapel-ChapelFcfToC.good

```diff
@@ -0,0 +1,2 @@
+passBadFnsToCChapel.chpl:18: error: unresolved call 'bar(chpl__fcf_type_void_void)'
+passBadFnsToCChapel.chpl:9: note: candidates are: bar(f: c_fn_ptr)
```

Comment:
I agree that this would be nice and think it could probably be done, but this is deep within the resolution machinery, so it's not immediately related to my change.  I could extend the futures that I currently have to request this though.